File: app/src/main/java/wmding/example/com/mylivetools/utils/SortDescendUtil.java
package wmding.example.com.mylivetools.utils;


import wmding.example.com.mylivetools.bean.ArticleDetailData;

/**
 * Created by CoderLengary
 */


public class SortDescendUtil {

    public static int sortArticleDetailData(ArticleDetailData articleDetailData, ArticleDetailData t1) {
        if (articleDetailData.getPublishTime() > t1.getPublishTime()){
            return -1;
        }else {
            return 1;
        }
    }

}

File: app/src/main/java/wmding/example/com/mylivetools/utils/SortDescendUtil.kt
package wmding.example.com.mylivetools.utils


import wmding.example.com.mylivetools.bean.ArticleDetailData

/**
 * Created by CoderLengary
 */


object SortDescendUtil {

    fun sortArticleDetailData(articleDetailData: ArticleDetailData, t1: ArticleDetailData): Int {
        return if (articleDetailData.publishTime > t1.publishTime) {
            -1
        } else {
            1
        }
    }

}
